src/components/dialogs/create-study-dialog.js

```
@@ -215,6 +258,12 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
         dispatch(setActiveDirectory(selectedDirectory?.elementUuid));
         dispatch(removeSelectedCase());
         setTriggerReset((oldVal) => !oldVal);
+        setFormatWithParameters([]);
+        setIsParamsCaseFileDisplayed(false);
+        if (tempCaseUuid !== null) {
+            deleteCase(tempCaseUuid).catch(() => setCreateStudyErr(''));
```

Why do you mask error ?

---

i will printed in the console